openassessment/xblock/xml.py

```diff
@@ -368,13 +428,6 @@ def parse_rubric_xml(rubric_root):
     """
     rubric_dict = dict()
 
-    # Rubric prompt
-    prompt_el = rubric_root.find('prompt')
```

Comment:
To add to what I wrote above, there is no thing as the rubric prompt. But the main text that is set onto the `prompt` field was being serialized inside the `rubric` element (which is why there is parsing logic here). I have moved it out to the root of the xml and this logic to the `_parse_prompts_xml ` function.